edx_rest_api_client/client.py

```diff
@@ -1,13 +1,24 @@
 import datetime
+import os
 
 import requests
+import requests.utils
 import slumber
 
 from edx_rest_api_client.auth import BearerAuth, JwtAuth, SuppliedJwtAuth
+from edx_rest_api_client.__version__ import __version__
 
 
 class EdxRestApiClient(slumber.API):
 
+    @classmethod
+    def user_agent(cls):
+        return "{} edx-rest-api-client/{} {}".format(
+            requests.utils.default_user_agent(),  # e.g. "requests/1.7.1"
```

Comment:
Note: The comment is a little confusing because "1.7.1" happens to be the __version__ in this PR.  Should this be `# e.g. "python-requests"` which is in the unit test, or is this typically something else?

Reply:
Yeah, I'm getting my version numbers crossed. The actual user agent that spits out of this thing looks like:

`python-requests/2.9.1 edx-rest-api-client/1.7.2 ecommerce`

I'll correct/add comments accordingly.